gdk4/src/auto/clipboard.rs

```diff
@@ -249,13 +244,8 @@ impl Clipboard {
         }
     }
 
-    //#[doc(alias = "gdk_clipboard_set_valist")]
-    //pub fn set_valist(&self, type_: glib::types::Type, args: /*Unknown conversion*//*Unimplemented*/Unsupported) {
-    //    unsafe { TODO: call ffi:gdk_clipboard_set_valist() }
-    //}
-
     #[doc(alias = "gdk_clipboard_set_value")]
-    pub fn set_value(&self, value: &glib::Value) {
+    pub fn set(&self, value: &glib::Value) {
```

Comment:
Why not `fn set(&self, value: &dyn ToValue)`?

Reply:
Maybe the same elsewhere

Reply:
Yes, that's much nicer